### ISSUE TITLE
Add "Privacy Policy" to "Page Type" location rules dropdown

### ISSUE THREAD

**Issue by crstauf** [AdvancedCustomFields/acf#688]
WordPress includes a screen to set the Privacy Policy for the site, which acts as a page type. Adding to the list of page types in the location rules should happen.

<img width="1107" alt="2022-08-15T202148" src="https://user-images.githubusercontent.com/4573033/184711720-f2e2e3ea-64e7-499f-8f6a-f891c4998703.png">

<img width="365" alt="2022-08-15T202618" src="https://user-images.githubusercontent.com/4573033/184712441-654cd194-6171-40c4-8e70-f8f2d738c3b6.png">

**Comment by CreativeDive** [comment]
This `is_privacy_policy();` could be also extended by:

```
is_date();
is_author();
is_archive();
is_post_type_archive();
is_tax();
is_category();
is_tag();
is_404();
is_search();
is_attachment();
```

**Comment by crstauf** [comment]
Many of those are already accessible through some other location rule, and none of them are actual Pages, so do not belong in the Page Type dropdown.

**Comment by CreativeDive** [comment]
> ... none of them are actual Pages ...

And that's why `Top Level Page`, `Parent Page`, `Child Page` are actual Pages? 

@crstauf Before you give a thumbs down, you should know what you're talking about. Thanks for that.

**Comment by CreativeDive** [comment]
@crstauf Yes, you've got it all. My mentioned additions are of course no kind of a page types. :clap:

**Comment by crstauf** [comment]
No worries. Sometimes it's hard to understand perspectives virtually. 🤗

**Comment by pedrosanta** [comment]
+1 for this.
